gems/pending/util/miq_logger_processor.rb

```diff
@@ -122,7 +122,7 @@ def cleanup_get_next_line
 
 class MiqLoggerLine < String
   def parts
-    @parts ||= self.class.split_raw_line(self).freeze
+    @parts ||= self.class.send(:split_raw_line, self).freeze
```

Comment:
We're adding a TODO in the other PR.  In a followup PR, we can drop the comment and just keep it public so there's no confusion.

https://github.com/ManageIQ/manageiq/pull/12041/files#diff-037377d58165f2739a6b9aeb4ef6976fR140